DevelopGitHub/GitHubExtension/NoteModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+public class NoteModel
```

Comment:
Sorry, that model doesn't exist in a final commit. Now model located in ...Notes.WebApi/ViewModels